core/src/main/scripts/migrate_db.py

```diff
@@ -0,0 +1,204 @@
+#! /usr/bin/env python
+
```

Comment:
Good point. We need to put a version number that the web app is compatible. This number should be accessible by Java code. Where should we put it @zheins @pieterlukasse ? Once we have that, we can display a message on the web page when the migration script needs to be run.